src/OpenLoco/src/Windows/MapWindow.cpp

```diff
@@ -112,6 +109,8 @@ namespace OpenLoco::Ui::Windows::MapWindow
 
     static WindowEventList events;
 
+    auto& lastMapWindowAttributes = LastMapWindow::getLastMapWindowAttributes();
```

Comment:
would prefer you don't get it like this with a global. Just get it where its used.